### PR TITLE
Add to cart

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added add to cart feature in trendy-t E-commerce. whenever the user clicks on add to cart button in Cart, It will add to the cart page, and also it will show the total price and discount. Users can increase the quantity of items in the cart Page.
Screenshot (289)

### CODE REVIEW

File: src/Pages/Cart-Page/Components/CartBill.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        totalPrice: 0,
+        totalDiscount: 0,
+      }
+    );
+
+  return (
+    <>
+      {cartdata.length === 0 ? (
+        false
+      ) : (
```
Comment (JantuDeb) [inline src/Pages/Cart-Page/Components/CartBill.jsx]: ```cartdata.length === 0 && <div className="cart__detail"> ...</div>```

**Review by JantuDeb — APPROVED** [review]

**Review by SuyashTiwariCoder — APPROVED** [review]
Code looks fine , folder structure is good , You can merge .

**Review by neetukumari4858 — APPROVED** [review]
UI looks good and code is readable.

File: src/Pages/Cart-Page/Components/CartBill.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Cart-Page/Components/CartBill.jsx]: Thankyou

**Comment by SaurabhChawda** [comment]
> Code looks fine , folder structure is good , You can merge .

I will change,ThankYou

**Comment by SaurabhChawda** [comment]
> UI looks good and code is readable.

ThankYou

**Pull request merged**